crates/hc_bundle/schema/happ-manifest.schema.json

```diff
@@ -26,6 +26,10 @@
       ],
       "description": "A description for the app"
     },
+    "membrane_proofs_deferred": {
+      "type": "boolean",
+      "description": "If true, membrane proofs are not provided at installation time, but later via app interface."
```

Comment:
I think this should be made a bit more accurate, e.g.:
"If true, any membrane proofs provided at installation time are ignored and need to be provided later via the app interface for the app to become functional"